src/main/java/com/n1analytics/paillier/EncryptedNumber.java

```diff
@@ -18,73 +18,7 @@
 import java.math.BigInteger;
 
 /**
- * Immutable class representing encrypted number and arithmetic operations that can be computed on the encrypted number.
```

Comment:
I've updated the high level documentation in 241cba3ff1395a573509067d2d56bb5185817a67